### ISSUE TITLE
NTR: Squalene low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8430]
New term request Squalene low in blood (elthat_200509142410):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000440
- label: Reduced circulating squalene concentration
- parents: [Abnormal circulating organic compound concentration (HP:0430071)](https://hpo.jax.org/app/browse/term/HP:0430071)
- pmid: 
- def: The concentration of squalene in the blood circulation is below the lower limit of normal.
- comment: Squalene, a highly unsaturated hydrocarbon from the triterpenoid family, is synthesized as a precursor for the synthesis of secondary metabolites such as sterols, hormones, or vitamins.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8430
- synonyms:
